src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1902,6 +1909,39 @@ private void StartAutoEvoPrediction()
 
         UpdateCompoundLastingTimes(compoundBalanceData, nightBalanceData, nominalStorage,
             specificStorages ?? throw new Exception("Special storages should have been calculated"));
+
+        // Handle process list
+        var processes = new List<TweakedProcess>();
+        var processStatistics = new List<ProcessSpeedInformation>();
+
+        ProcessSystem.ComputeActiveProcessList(editedMicrobeOrganelles, ref processes);
+
+        float consumptionProductionRatio = energyBalance.TotalConsumption / energyBalance.TotalProduction;
+
+        foreach (var process in processes!)
+        {
+            var singleProcess = ProcessSystem.CalculateProcessMaximumSpeed(process, biome, CompoundAmountType.Current);
```

Comment:
The compound amounts here may or may not be needed to be hooked up to the time of day selector button in the editor. Though I'm not fully sure so this needs more investigation (I'd appreciate anyone else trying to look into this before I have to).

Reply:
Was this checked?